aptos-move/framework/move-stdlib/sources/hash.move

```diff
@@ -3,12 +3,20 @@
 /// The functions in this module are natively declared both in the Move runtime
 /// as in the Move prover's prelude.
 module std::hash {
+    use std::bcs;
+
     // TODO: move sip_hash into aptos_framework
-    native public fun sip_hash<MoveValue>(v: &MoveValue): u64;
-    native public fun sha2_256(data: vector<u8>): vector<u8>;
-    native public fun sha3_256(data: vector<u8>): vector<u8>;
+    public fun sip_hash<MoveValue>(v: &MoveValue): u64 {
+        let bytes = bcs::to_bytes(v);
+
+        sip_hash_internal(bytes)
+    }
 
     spec sip_hash { // TODO: temporary mockup.
         pragma opaque;
     }
+
+    native fun sip_hash_internal(bytes: vector<u8>): u64;
```

Comment:
I would suggest we make this public and name it sip_hash, just to be consistent with other crypto functions.

Reply:
Ah! Okay.